### PR TITLE
use transform

### PR DESCRIPTION
**Pull request by yoshuawuyts**
Semver major. Replaces `--use` with `--transform`. Tests pass, but not sure if
`package.json` detection works. Did we ever get that to work? Thanks!

### CODE REVIEW

**Comment by cjhowe7** [comment]
As I mentioned in #136, this is failing with `postcss-prefix@2.1.0`. I definitely don't suggest merging this, at least without changing the `postcss-prefix` semver!

**Comment by yoshuawuyts** [comment]
@cjhowe7 could you maybs give this another review? Thanks! :grin:

File: test/fixtures/transform-package/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      [
+        "sheetify-cssnext",
+        {
+          "sourcemap": false
+        }
+      ]
+    ]
+  },
+  "dependencies": {
+    "sheetify": "file:../../..",
```
Comment (cjhowe7) [inline test/fixtures/transform-package/package.json]: Not so sure these dependencies are necessary.

File: test/transform-package.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    const expath = path.join(__dirname, 'fixtures/transform-package/expected.css')
+    const expected = fs.readFileSync(expath, 'utf8').trim()
+
+    const ws = concat(function (buf) {
+      const res = String(buf).trim()
+      t.equal(res, expected, 'CSS was transformed')
+    })
+
+    const bOpts = { browserField: false }
```
Comment (cjhowe7) [inline test/transform-package.js]: This is probably unnecessary

File: test/transform-package.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    browserify(bpath, bOpts)
+      .transform(sheetify)
+      .transform(function (file) {
+        return through(function (buf, enc, next) {
+          const str = buf.toString('utf8')
+          this.push(str.replace(/sheetify\/insert/g, 'insert-css'))
+          next()
+        })
+      })
+      .plugin('css-extract', { out: outFn })
```
Comment (cjhowe7) [inline test/transform-package.js]: Well, this is definitely not supposed to be here for this test. Woops.

**Comment by yoshuawuyts** [comment]
Thanks!y

File: test/transform-package.js (hunk 3 above)
Comment (cjhowe7) [reply test/transform-package.js]: No, wait, my bad. This is fine. I guess I confused this with `sheetify-cssnext`.

File: test/fixtures/transform-package/package.json (hunk 1 above)
Comment (cjhowe7) [reply test/fixtures/transform-package/package.json]: Confirmed they are unnecessary. I will PR shortly to remove this.

**Comment by cjhowe7** [comment]
I must have been tired when I did that review, so it should actually be working fine. If you want, I opened a pull request (https://github.com/stackcss/sheetify/pull/137) to remove some unnecessary options from the tests to keep them more minimal.

**Comment by yoshuawuyts** [comment]
`v7.0.0` :tada:

**Pull request merged**